group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -338,7 +338,6 @@ public void testConsumerGroupMemberEpochValidation() {
         ConsumerGroupMember member = new ConsumerGroupMember.Builder(memberId)
             .setMemberEpoch(100)
             .setPreviousMemberEpoch(99)
-            .setTargetMemberEpoch(100)
```

Comment:
should we explicitly set state here? I think it is stable by default but the other builders added it.

Reply:
I think that it makes sense to add it. I reviewed all the cases in the file and added where it makes sense.